Dockerfile.alpine

```diff
@@ -54,9 +54,21 @@ EXPOSE 1883 8883 8080 44053 4369 8888 \
        9100 9101 9102 9103 9104 9105 9106 9107 9108 9109
 
 
-VOLUME ["/vernemq/log", "/vernemq/data", "/vernemq/etc"]
```

Comment:
Can you explain why you've removed the `/vernemq/etc` volume?

Reply:
- Docker container runtime modifies permissions for folders listed as volumes when spinning up a  container if no special mount point is defined.
- Helm chart does not mount anything to `/vernemq/etc`.
- This creates issues when running as random user.

Reply:
With your addition would mounting a custom `/vernemq/etc/vernemq.conf.local` still work?

Reply:
@dergraf it shouldn't make a difference, and @blazdivjak is probably right, it shouldn't have been there in the first place most likely. You can still mount that file from within docker run or k8s even without the volume statement.

Reply:
Ok, thanks. Does this mean that we can get rid of `log` and `data` too?